docs/styles.md

```diff
@@ -0,0 +1,34 @@
+# Styling content for RAISE
+
+In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. 
+
+RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below.
+
+## Style Classes 
+
+### Indent (os-raise-indent)
+Add a padding to the left side of an element. If the element already includes a left padding (like an ordered list) the indent will get set to exactly 32px
```

Comment:
This `32px` is actually slightly incorrect. You can refer to [this documentation](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#ems_and_rems) as far as why, but basically if you use `2rem` it will scale with the root font size.